### ISSUE TITLE
There is no "contao-components/ace" asset package

### ISSUE THREAD

**Issue by fritzmg** [contao/managed-edition#30]
When installing Contao 4.5.1 from the [zip archive](https://download.contao.org/4.5.1/zip), the following error will occur when trying to create a new page layout:
```
Symfony\Component\Asset\Exception\InvalidArgumentException:
There is no "contao-components/ace" asset package.

  at vendor\symfony\symfony\src\Symfony\Component\Asset\Packages.php:78
  at Symfony\Component\Asset\Packages->getPackage('contao-components/ace')
     (vendor\symfony\symfony\src\Symfony\Component\Asset\Packages.php:109)
  at Symfony\Component\Asset\Packages->getUrl('js/ace.js', 'contao-components/ace')
     (vendor\contao\core-bundle\src\Resources\contao\library\Contao\Template.php:368)
  at Contao\Template->asset('js/ace.js', 'contao-components/ace')
     (vendor\contao\core-bundle\src\Resources\contao\templates\backend\be_ace.html5:8)
  at include('vendor/contao/core-bundle/src/Resources/contao/templates/backend/be_ace.html5')
     (vendor\contao\core-bundle\src\Resources\contao\library\Contao\TemplateInheritance.php:98)
  at Contao\Template->inherit()
     (vendor\contao\core-bundle\src\Resources\contao\library\Contao\Template.php:285)
  at Contao\Template->parse()
     (vendor\contao\core-bundle\src\Resources\contao\classes\BackendTemplate.php:34)
  at Contao\BackendTemplate->parse()
     (vendor\contao\core-bundle\src\Resources\contao\classes\DataContainer.php:600)
  at Contao\DataContainer->row('{title_legend},name;{header_legend},rows,[rows],headerHeight,[EOF];{column_legend},cols,[cols],widthLeft,[EOF];{sections_legend:hide},sections;{webfonts_legend:hide},webfonts;{style_legend},framework,stylesheet,external,loadingOrder,combineScripts;{picturefill_legend:hide},picturefill;{feed_legend},newsfeeds,calendarfeeds;{modules_legend},modules;{jquery_legend:hide},addJQuery;{mootools_legend:hide},addMooTools;{script_legend},scripts,analytics,externalJs,script;{static_legend:hide},static;{expert_legend:hide},template,doctype,viewport,titleTag,cssClass,onload,head')
     (vendor\contao\core-bundle\src\Resources\contao\drivers\DC_Table.php:1989)
  at Contao\DC_Table->edit()
     (vendor\contao\core-bundle\src\Resources\contao\classes\Backend.php:629)
  at Contao\Backend->getBackendModule('themes', null)
     (vendor\contao\core-bundle\src\Resources\contao\controllers\BackendMain.php:134)
  at Contao\BackendMain->run()
     (vendor\contao\core-bundle\src\Controller\BackendController.php:50)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php:151)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php:68)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Kernel.php:202)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (web\app_dev.php:65)
```
This error does not occur when Contao 4.5.1 is installed (or updated to) via composer.

**Comment by leofeyer** [comment]
Jeez, we really need to stop distributing .zip files. :unamused:

**Comment by leofeyer** [comment]
This is due to the `Versions` class not being generated upon build.

**Comment by leofeyer** [comment]
Fixed in contao/release-helper@f579b70cbe99df16eb93bd880547b41ebebab5e1.
